datasets/humor-patterns.yaml

```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns

```

Comment:
Please add a little more description to the name that'd help someone know what this data is.